crates/sargon-uniffi/src/keys_collector/derivation_purpose.rs

```diff
@@ -23,6 +23,10 @@ pub enum DerivationPurpose {
     /// for identity MFA
     SecurifyingPersona,
 
+    /// When applying a security shield to accounts and personas mixed, initiates keys collection
+    /// for account MFA
+    SecurifyingAccountsAndPersonas,
```

Comment:
Yes if you only security account or only security personas those other are used. 

But host will most likely map all variants to same Crowdin key anyway...

Reply:
In the host API, it will send a list of AccountOrPersona right? and then in Sargon you map to specific purpose based on the input, I think I've seen this logic in this PR.